container-assets/entrypoint

```diff
@@ -42,6 +42,9 @@ else
   su postgres -c "initdb -D ${APPLIANCE_PG_DATA}"
   test $? -ne 0 && echo "!! Failed to initdb" && exit 1
 
+  mkdir /var/run/postgresql
+  chown postgres:postgres /var/run/postgresql
```

Comment:
If someone maps in a volume for postgresql, will this cause any problems.  Should it be mkdir -p ?

Reply:
This is different from the postgres data directory.  But I moved it up so that it always runs and added `-p` for safety.